src/serlo-editor/math/editor.tsx

```diff
@@ -260,16 +280,24 @@ export function MathEditor(props: MathEditorProps) {
   }
 
   function renderOverlayPortal() {
-    const children = (
+    return (
       <div className="fixed bottom-0 z-50 rounded-t-xl bg-editor-primary-100 p-3 shadow-menu">
-        <p className="mr-0.5 mt-1 text-right text-sm font-bold text-gray-600">
-          {hasError ? mathStrings.onlyLatex : mathStrings.latexEditorTitle}
-        </p>
-        {!useVisualEditor && (
+        <div className="flex items-center justify-between">
+          <p className="mr-0.5 mt-1 text-right text-sm font-bold text-gray-600">
+            {hasError ? mathStrings.onlyLatex : mathStrings.latexEditorTitle}
+          </p>
+          <button
+            onClick={() => props.onMoveOutRight({ closeThroughModal: true })}
+            className="mr-0.5 mt-1 text-sm font-bold text-gray-600"
+            data-qa="plugin-math-close-formula-editor"
+          >
+            x
+          </button>
```

Comment:
Sorry, a11y veto here :) 

If you really want to add this button:
- please add hover state
- add [hidden label](https://tailwindcss.com/docs/screen-readers) for screen readers

Also for consistency I'd prefer to use an icon here instead of `x` like we do for the other close buttons.